.ci/containers/membership-checker/utils.go

```diff
@@ -38,7 +43,7 @@ func requestCall(url, method, credentials string, result interface{}, body inter
 }
 
 func readFile(filename string) (string, error) {
```

Comment:
Would it make sense to embed the file as a string directly, rather than embed a filesystem and then process it? Per https://pkg.go.dev/embed#hdr-File_Systems:

> For embedding a single file, a variable of type string or []byte is often best. The FS type enables embedding a tree of files, such as a directory of static web server content, as in the example above.

Reply:
I think it make more sense that way. Just updated it.